tools/report.sh

```diff
@@ -333,6 +404,15 @@ for POD in $PODS; do
   fi
 done
 
+echo "kafka"
+mkdir -p "$OUT_DIR"/reports/kafka/
+SERVICE=$($KUBE_CLIENT -n "$NAMESPACE" get service  -o name -l "app.kubernetes.io/name=kafka" | grep bootstrap | head -n1) && readonly SERVICE
+BOOTSTRAP_SERVER=$($KUBE_CLIENT get kafka "$CLUSTER" -o=jsonpath='{.status.listeners[?(@.type=="plain")].bootstrapServers}' -n "$NAMESPACE") && readonly BOOTSTRAP_SERVER
```

Comment:
a `plain` listener could not exist at all, and the cluster having a TLS based one only. In such a case you would need to get cluster CA certificate to connect (at least for encryption).